package.json

```diff
@@ -35,12 +35,7 @@
     "react": "^16.12.0",
     "react-bootstrap": "^0.31.0",
     "react-dom": "^16.12.0",
-<<<<<<< HEAD
     "react-google-login": "^5.1.24",
-=======
-    "react-google-login": "^5.1.22",
-    "react-router": "^5.2.0",
```

Comment:
isn't the "react-router": "^5.2.0" required ?

Reply:
I think so, I will remove this branch completely since we mainly used this to submit the MVP